scripts/create_deployment_database.sh

```diff
@@ -28,7 +28,7 @@ else
   echo "Logging only enabled..."
 fi
 
-DB_ADMIN_PASSWORD=$(openssl rand -base64 32)
+DB_ADMIN_PASSWORD=$(cat /dev/urandom | tr -dc 'A-Za-z0-9!' | head -c 13 ; echo ` `)
```

Comment:
what's the purpose of this switch?

Reply:
Sorry - it's detailed in this PR: https://github.com/Zenysis/Harmony/pull/90 - previous algorithm was generating characters that confuses hasura